styles.css

```diff
@@ -107,3 +107,71 @@ input {
 .right-input {
   box-sizing: calc(border-box / 2);
 }
+
+.start-btn {
+  background-color: white;
+  color: #2B2733;
+}
+
+.btn-start {
+display: flex;
+justify-content: flex-end;
+padding: 10px 0;
+}
+
+@media screen and (max-width: 949px) {
+  .icon,
+  .btn-text {
+    font-size: 3em;
+  }
+
+  .activities {
+    display: flex;
+    display: grid;
+    flex-direction: column;
+    grid-template-columns: 1fr;
+  }
+
+  .past-activities {
+    border-top: solid;
+    border-left: none;
+    border-color: #46424D;
+    padding: 2%;
+  }
+
+  .start-btn,
+  .btn-container {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    justify-content: space-around;
+  }
+
+  .study-btn,
+  .meditate-btn,
+  .exercise-btn {
+    display: grid;
+    margin: 0 20%;
+    margin-bottom: 2em;
```

Comment:
To get the button text & icons centered, I think we'll be looking at align-items or justify-items in this btn styling.